index.bs

```diff
@@ -129,25 +134,65 @@ The {{Accelerometer/z!!attribute}} attribute of the {{Accelerometer}}
 interface represents the <a>acceleration</a> along Z-axis.
 This attribute returns [=map/value=] for [=latest reading=]["z"] [=map/entry=].
 
-### Accelerometer.includesGravity ### {#accelerometer-includesGravity}
+The LinearAccelerometer Interface {#linear-accelerometer-interface}
```

Comment:
Looks like  `LinearAccelerometer` term is used https://developer.android.com/guide/topics/sensors/sensors_motion.html#sensors-motion-linear